cpp/ql/lib/change-notes/2022-03-14-flow-state-barriers.md

```diff
@@ -1,4 +1,4 @@
 ---
 category: feature
 ---
-* The data flow and taint tracking libraries have been extended with versions of `isBarrierIn`, `isBarrierOut`, and `isBarrierGuard`, respectively `isSanitizerIn`, `isSanitizerOut`, and `isSanitizerGuard`, that support flow states.
+* The data flow and taint tracking libraries have been extended with versions of `isBarrierGuard`, respectively `isSanitizerGuard`, that support flow states.
```

Comment:
I didn't keep track when the latest release was, but that might already have deleted this change note and the others like this..